pkg/sdk/testint/network_policies_gen_integration_test.go

```diff
@@ -85,6 +108,90 @@ func TestInt_NetworkPolicies(t *testing.T) {
 		assert.Equal(t, 1, np.EntriesInBlockedIpList)
 	})
 
+	t.Run("Alter - set allowed network rule list", func(t *testing.T) {
+		allowedNetworkRule := createNetworkRuleHandle(t, client)
+
+		req := defaultCreateRequest()
+		err, dropNetworkPolicy := createNetworkPolicy(t, client, req)
+		require.NoError(t, err)
+		t.Cleanup(dropNetworkPolicy)
+
+		err = client.NetworkPolicies.Alter(ctx, sdk.NewAlterNetworkPolicyRequest(req.GetName()).
+			WithSet(sdk.NewNetworkPolicySetRequest().WithAllowedNetworkRuleList([]sdk.SchemaObjectIdentifier{allowedNetworkRule})))
```

Comment:
can we add/extend this test to have two items? We'll know the SQL is generating correctly with commas and we're testing that it works on the Snowflake side.